README.md

```diff
@@ -12,7 +12,7 @@ DKPro Argumentation is available from Maven Central. For working with the UIMA t
 <dependency>
     <groupId>de.tudarmstadt.ukp.dkpro.argumentation.types</groupId>
     <artifactId>de.tudarmstadt.ukp.dkpro.argumentation.types</artifactId>
-    <version>0.0.2</version>
+    <version>0.0.3-SNAPSHOT</version>
```

Comment:
Sorry for not being clear. This README file is currently the only documentation for end-users who want to use the library. We released the version 0.0.2 on Maven Central and this README shows the users how to incorporate the latest stable released version into their projects. The current development version is 0.0.3-SNAPSHOT (which is not yet on Maven Central). Thus changing the user guides to pointing to release which doesn't exist is confusing.